NetCord/Application.cs

```diff
@@ -65,13 +44,177 @@ public Application(JsonModels.JsonApplication jsonModel, RestClient client) : ba
+    /// The partial <see cref="RestGuild"/> object of the application's associated guild.
+    /// </summary>
+    public RestGuild? Guild { get; }
+
+    /// <summary>
+    /// The ID of the Game SKU created, if one exists. Is <see langword="null"/> if the application is not a game sold on Discord.
+    /// </summary>
+    public ulong? PrimarySkuId => _jsonModel.PrimarySkuId;
+
+    /// <summary>
+    /// The URL slug that links to an application's store page. Is <see langword="null"/> if the application is not a game sold on Discord.
+    /// </summary>
+    public string? Slug => _jsonModel.Slug;
+
+    /// <summary>
+    /// The cover image hash of the application's default rich presence invite.
+    /// </summary>
+    public string? CoverImageHash => _jsonModel.CoverImageHash;
+
+    /// <summary>
+    /// The application's public flags.
+    /// </summary>
+    public ApplicationFlags? Flags => _jsonModel.Flags;
+
+    /// <summary>
+    /// The approximate number of guilds the application has been added to.
+    /// </summary>
+    public int? ApproximateGuildCount => _jsonModel.ApproximateGuildCount;
+
+    /// <summary>
+    /// The approximate number of users that have installed the application.
+    /// </summary>
+    public int? ApproximateUserInstallCount => _jsonModel.ApproximateUserInstallCount;
+
+    /// <summary>
+    /// The application's redirect URI list.
+    /// </summary>
+    public IReadOnlyList<string>? RedirectUris => _jsonModel.RedirectUris;
+
+    /// <summary>
+    /// The application's interactions endpoint URL.
+    /// </summary>
+    public string? InteractionsEndpointUrl => _jsonModel.InteractionsEndpointUrl;
+
+    /// <summary>
+    /// The application's role connection verification URL.
+    /// </summary>
+    public string? RoleConnectionsVerificationUrl => _jsonModel.RoleConnectionsVerificationUrl;
+
+    /// <summary>
+    /// The application's tag list, describing its content and functionality. Max of 5 tags.
+    /// </summary>
+    public IReadOnlyList<string>? Tags => _jsonModel.Tags;
+
+    /// <summary>
+    /// The application's default in-app authorization URL. Is <see langword="null"/> if disabled.
+    /// </summary>
+    public ApplicationInstallParams? InstallParams { get; }
+
+    /// <summary>
+    /// The default scopes and permissions for each supported installation context.
+    /// </summary>
+    public IReadOnlyDictionary<ApplicationIntegrationType, ApplicationIntegrationTypeConfiguration>? IntegrationTypesConfiguration { get; }
+
+    /// <summary>
+    /// The application's default customization URL. Is <see langword="null"/> if disabled.
+    /// </summary>
+    public string? CustomInstallUrl => _jsonModel.CustomInstallUrl;
+
+    /// <summary>
+    /// Gets the <see cref="ImageUrl"/> of the application's icon.
+    /// </summary>
+    /// <param name="format">The format of the returned <see cref="ImageUrl"/>. Defaults to <see cref="ImageFormat.Png"/>.</param>
```

Comment:
I don't think the format defaults to something as it's required. Same for other methods.